### ISSUE TITLE
Skeletal Sword

### ISSUE THREAD

**Issue by Sagisolar** [Tslat/Advent-Of-Ascension#164]
## HISTORY ##

The Skeletal Sword has had a rough year, in the beginning of the year it was the best sword in the mod at 31 damage with no drawback. The Tslat versions weren't kind to it however. Having its damage lowered to 24. Not sure if Scimi intended this but he accidentally made it do the same damage as the Vulcammer, that would be fine but the Vulcammer has a special ability, which means its a direct upgrade to it, even had the same durability.

Now with 3.0.
Changed to 23 damage and basically got screwed over by the Scythes, they do the same damage while having abilities, sure they requires energy, but you can use the Ghoulish Armor. Not only that Scythes attack faster than swords.
The reason why this is a issue is because The Skeletal Army will probably become one of the hardest bosses in the game (Shout out to Show Staff nerf). In fact, I'm sure it would be harder than the AC bosses. 

## CHANGE ##

23 damage > 26 damage.

Harvester Sword
Kinda bad after being nerfed in the Tslat versions.

25 damage > 29 damage.

**Comment by Cruuk** [comment]
No. Swords are fine as they are. You do know that Skeletal sword still ranks as basically the overall best sword right? (Harvester is more of an alternative than an upgrade to me).

Consider the DPS of all the swords: Sword damage x 1.6. Skeletal and Harvester get 36.8 and 40 DPS respectively. 40, that's the default for endgame DPS. You may say Megagun is better cause it gets 40 DPS, but no, you can pull out the swords way faster than Megagun and swords have many things going for them like critical hits, battle rage, and the strength effect. Oh yeah and sharpness.

Scythes are overall better because they're ENDGAME, MORE ENDGAME THAN SKELETAL. However, consider that Skeletal has slightly higher DPS than the scythes (other than Pluton), Scythes just best Skeletal in range and ability while Skeletal is *slightly* better in raw damage, also I don't think Scythes can vanilla crit.
Also IIRC Scythes aren't affected by Recharge and Ghoulish armor.

Also if you buff those swords by that amount greatblades will get fucked.

Also considering certain bosses becoming way harder due to weapon nerfs: What's stopping Scimi from nerfing the boss instead? I'm not saying to nerf the Skeletal army but why does the weapon have to get buffed when the mob/boss can be nerfed instead?

I do partially agree about the decision to make multiple swords do 24 damage in 1.7.10, at least make Skeletal do like 25 or whatever
I don't care about 1.7.10 anymore so it no longer concerns me.

Do research before making a balance suggestion next time please.

**Comment by Sagisolar** [comment]
How do you remove a post?

**Comment by Cruuk** [comment]
> 
> 
> How do you remove a post?

I think only Scimi can, ask him I guess.
